lib/Utils/Architectures.pm

```diff
@@ -126,4 +128,30 @@ sub is_ppc64le {
     return check_var('ARCH', 'ppc64le');
 }
 
+=head2 is_orthos_machine
+
+ is_orthos_machine();
+
+Returns C<true if macine FQDN has arch.suse.de suffix>.
+
+=cut
+sub is_orthos_machine {
+    my $sut_fqdn = get_var('SUT_IP', 'nosutip');
+    return 1 if $sut_fqdn =~ /(arch\.suse\.de)/im;
+    return 0;
+}
+
+=head2 is_valid_machine
```

Comment:
this is a bit too generic. Every machine that we run tests on can be considered "valid". I wonder, what is the *actual* requirement?

Reply:
Before introducing arm64 ipmi backend in boot_from_pxe module, only x86_64 machines boot from pxe and they use "esc" to bring the test into "qa-net-boot". Arm64 ones use different key 'c' and also different set of command lines to download linux/initrd and boot from there, that is why there is "        if (is_remote_backend && is_aarch64 && is_valid_machine)" in general. And I do not think all arm64 machines need this, so I put "is_valid_machine" there as well to filter out only those involved, it originally only covers qa-apac2 and orthos, now extends to openQA. I do not think it covers everything, at least, it only covers DE and ASIA partially.

Reply:
ok, but can you then please use a less generic name than "valid"

Reply:
Done.

Reply:
sorry but I do not see "covered" better than "valid". also see below

Reply:
I think it is enough. If you think there is better one, please suggest your preferred subroutine name. Please say your favorite name.